### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by bug-smith**
Take 1
Screenshot 2023-09-05 at 10 49 22 AM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <h1>Coding Form!</h1>
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" />
+
+      <p>Why is coding cool?</p>
```
Comment (shawnkost) [inline html-forms/index.html]: This should be a `label` that will need to be linked to your `select` via a `for` attribute

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+      <p>
+        <select id="dropdown" name="why is coding cool?">
```
Comment (shawnkost) [inline html-forms/index.html]: Use hypens to seperate words rather than spaces. `why-is-coding-cool`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <input type="radio" id="CSS" name="radio1" />
+        <label for="CSS">CSS</label>
+        <input type="radio" id="JavaScript" name="radio1" />
+        <label for="JavaScript">JavaScript</label>
+      </fieldset>
+
+      <p>Would you like to sign up for more information?</p>
+
+      <fieldset>
+        <input type="checkbox" id="SignMeUp" />
```
Comment (shawnkost) [inline html-forms/index.html]: `id` naming convention prefers `kebab-cased` rather than `camelCased`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <label for="SignMeUp">Sign me up!</label>
+      </fieldset>
+
+      <p>Comments</p>
+
+      <label for="comments"></label>
+      <textarea id="comments" name="comments" cols="20" rows="7"></textarea>
+      <p>
+        <button type="submit">Submit</button>
+        <label for="submit"></label>
```
Comment (shawnkost) [inline html-forms/index.html]: Since there's no text inside this `label`, it's not needed. We only need to include them if we have text associated with it

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Bug!

**Comment by bug-smith** [comment]
Screenshot 2023-09-05 at 11 31 33 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**